### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by username-pass** [username-pass/bookmarklets#17] [Polls]
## To everyone who has just recently come to this repo, and decided to stick around...

Welcome!

**Comment by hokjda** [comment]
i was researching your profile cause of inject0r dev team application

**Reply by GhostieKoto** [reply]
Bookmarklets don't work. They were blocked

**Comment by Zacky2613** [comment]
Found on github Explore tab

**Reply by username-pass** [reply]
cool! How can I improve this?

**Comment by XenkoHD** [comment]
Heyo, my district also blocked some bookmarklets. Could you please share your bypass? I really wanna be able to execute them for many reasons.

**Reply by username-pass** [reply]
the "bypass" only works as a way to execute code in general unfortunately. I haven't worked on this in a while... Essentially, it's a site that runs the bookmarklet, so you can run non site specific code, but you cannot run things like darkmode, as it does not run in any specific site.

**Reply by XenkoHD** [reply]
I see, well, could you then post the website but require a proxy to access it so it can't be seen? Or just post the website? Please.

**Reply by username-pass** [reply]
this should work, it is the same bml, but wrapped in a data url. it just bypasses javascript:// urls, but does have the above mentioned limitations.
```
data:text/html,<script>fetch("https://raw.githubusercontent.com/username-pass/bookmarklets/main/all-bookmarklets.js").then(function(response){response.text().then(function(text) {eval(text);});});</script>
```

**Reply by ONI234** [reply]
what is the pass for the injector

**Reply by ONI234** [reply]
and the username

**Reply by username-pass** [reply]
@ONI234 This is not Inject0r, so I will not give you that information here, as I do not want to set that precedent, and have this become a place for people to ask for tokens. Either ask in the Inject0r discord, or Github, but not this project. Here's the github for Inject0r [https://github.com/Platinome/inject0r](https://github.com/Platinome/inject0r), and the discord [https://discord.gg/GSqDrERGze](https://discord.gg/GSqDrERGze)

**Comment by XenkoHD** [comment]
the thing only works in the data:text/html,<script>fetch("https://raw.githubusercontent.com/username-pass/bookmarklets/main/all-bookmarklets.js").then(function(response){response.text().then(function(text) {eval(text);});});</script> site

**Comment by TheVoiceModifier** [comment]
I wanted to do fun things that my classmates can't
